syntax = "proto3";

message Request {
  int32 code = 1;
}

message MemoryUsageResponse {
  int32 usage = 1;
}

service MonitorC {
  rpc GetMemoryUsage (Request) returns (MemoryUsageResponse) {}
}
